File: we_1_8_from_uncertainty_to_a_reported_error.m
% Integrated Navigation Systems — Chapter 1: Introduction to Integrated Navigation and PNT
% Worked Example 1.8 — From uncertainty to a reported error bound
% Companion code (educational reference implementation; see the book for context).

sigmaH = 6.0;              % horizontal 1-sigma (m)
R95 = 2.45 * sigmaH;      % ~95% radius, circular approximation
fprintf('R95 = %.1f m\n', R95);   % -> 14.7 m

File: we_1_8_from_uncertainty_to_a_reported_error.py
# Integrated Navigation Systems — Chapter 1: Introduction to Integrated Navigation and PNT
# Worked Example 1.8 — From uncertainty to a reported error bound
# Companion code (educational reference implementation; see the book for context).

sigmaH = 6.0              # horizontal 1-sigma (m)
R95 = 2.45 * sigmaH       # ~95% radius, circular approximation
print(f'R95 = {R95:.1f} m')   # -> 14.7 m
